playground/src/example.ml

```diff
@@ -32,4 +31,22 @@ let rec fib_par n d =
 let () =
   let res = fib_par n num_domains in
   Printf.printf "fib(%d) = %d\n" n res
+
+(*
+  By the way, a much better, single-threaded implementation that calculates
+  the Fibonacci sequence is this:
+
+  let fib n =
+    let rec loop n =
+      if n < 1 then (0, 1)
+      else
+        let x, y = loop (n-1) in
+        y, (x+y)
+    in
+    snd (loop n);;
```

Comment:
Isn't a tail-recursive version preferable? Here is how I like to write it:
```ocaml
let rec fib m n i = if i < 1 then m else fib n (n + m) (i - 1)
let fib = fib 0 1
```
But there's so many good ways to write that kind of stuff :-)

Reply:
changed example to this variant